File: CS/TaskManager/TasksCode/Task.cs
using System;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
namespace TaskManager.TaskDatabase
{

    public partial class Task
    {
        public Task(Session session) : base(session) { }
        public Task() : base(Session.DefaultSession) { }
        public override void AfterConstruction() { base.AfterConstruction(); }
    }

}

File: VB/TaskManager/TasksCode/Task.vb
Imports System
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Namespace TaskManager.TaskDatabase

    Partial Public Class Task
        Public Sub New(ByVal session As Session)
            MyBase.New(session)
        End Sub
        Public Sub New()
            MyBase.New(Session.DefaultSession)
        End Sub
        Public Overrides Sub AfterConstruction()
            MyBase.AfterConstruction()
        End Sub
    End Class

End Namespace
